templates/registration/login.html

```diff
@@ -34,7 +34,11 @@
       <button> Submit</button>
   
   </form>
+<<<<<<< HEAD
```

Comment:
@christinali can u fix this? didn't show up as merge conflict for some reason

Reply:
on it